h/services/search_index/_queue.py

```diff
@@ -188,3 +186,10 @@ def _get_annotations_from_es(self, annotation_ids):
             hit["_source"]["updated"] = updated
 
         return {hit["_id"]: hit["_source"] for hit in hits}
+
+    @staticmethod
+    def _datetime_from_seconds(seconds):
```

Comment:
Changed it to `_datetime_at` and added a docstring. I went for `delta_seconds` since that's what Python calls it (`timedelta(seconds=delta_seconds)`):

```python
@staticmethod
def _datetime_at(delta_seconds):
    """Return the datetime at delta_seconds seconds from now."""
    if not delta_seconds:
        return None
    return datetime.utcnow() + timedelta(seconds=delta_seconds)
```